engine/engine/src/engine.cpp

```diff
@@ -426,11 +426,12 @@ namespace dmEngine
             const char* mountstr = "";
 #if defined(__NX__)
             mountstr = "data:/";
+#endif
             // there's no way to check for a named mount, and it will assert
             // So we'll only enter here if it's set on this platform
-            if (dmSys::GetEnv("DM_MOUNT_HOST") != 0)
-                mountstr = "host:/";
-#endif
+            if (dmSys::GetEnv("DM_HOSTFS") != 0)
+                mountstr = dmSys::GetEnv("DM_HOSTFS");
+
             dmSnPrintf(p1, sizeof(p1), "%sgame.projectc", mountstr);
```

Comment:
If DM_HOSTFS is set, we will look for a project there. Good for running unit tests on the device, but keeping the data on the host PC